cache_toolbox/core.py

```diff
@@ -108,8 +108,14 @@ def instance_key(model, instance_or_pk):
     Returns the cache key for this (model, instance) pair.
     """
 
+    try:
+        model_name = model._meta.model_name
+    except AttributeError:
+        # Django version <1.6
```

Comment:
Pass is only used to mark an empty block as being empty. This isn't an empty block.